src/base/preferences.cpp

```diff
@@ -308,6 +308,16 @@ void Preferences::setMailNotificationEnabled(bool enabled)
     setValue("Preferences/MailNotification/enabled", enabled);
 }
 
+QString Preferences::getMailNotificationSender() const
+{
+    return value("Preferences/MailNotification/sender", "qBittorrent_notification@example.com").toString();
```

Comment:
Why not "notification@qbittorrent.org" as it was before?

Reply:
@example.com is more encouraging to replace it, I guess because of that.

Reply:
I think using `qbittorrent.org` domain is not a good idea, the mail clearly doesn't come from there.
also https://serverfault.com/questions/333548/where-does-email-sent-to-example-com-go/333552